src/plugins/score-plugin-protocols/Protocols/MIDI/MIDIProtocolSettingsWidget.cpp

```diff
@@ -31,14 +31,20 @@ MIDIInputSettingsWidget::MIDIInputSettingsWidget(QWidget* parent)
   m_name = new State::AddressFragmentLineEdit{this};
   checkForChanges(m_name);
   m_createWhole = new QCheckBox{tr("Create whole tree"), this};
+  m_virtualPort = new QCheckBox{tr("Virtual Port"), this};
 
   auto lay = new QFormLayout;
   lay->addRow(tr("Name"), m_name);
   lay->addRow(m_createWhole);
+  lay->addRow(m_virtualPort);
 
   setLayout(lay);
   m_createWhole->setChecked(false);
-  m_createWhole->setEnabled(true);
+  m_createWhole->setEnabled(false);
+  m_virtualPort->setChecked(false);
+#if !__WIN32
```

Comment:
I don't think this works as "enabled" is the default state, it should be 

```
#if defined(_WIN32) // note the single underscore
  m_virtualPort->setEnabled(false);
#endif
```

instead (with maybe a comment to say that this is because win32 does not support that)

Reply:
Done